stable/kube-state-metrics/Chart.yaml

```diff
@@ -5,11 +5,13 @@ keywords:
 - metric
 - monitoring
 - prometheus
-version: 0.3.1
-appVersion: 1.0.1
+version: 0.3.2
+appVersion: 1.1.0
```

Comment:
The container image version in the `values.yaml` file has not increased. So, it is delivering version 1.0.1. To update the app version the container image should also be updated to one with that app version.

Reply:
my mistake, thank you.I also update the values of `version` and `appVersion`